tests/unit/functions/github.test.ts

```
@@ -39,8 +48,13 @@ describe("createGithubTeam", () => {
     vi.clearAllMocks();
     mockOctokit = {
       request: vi.fn(() => {}),
+      paginate: {
+        iterator: vi.fn(),
+      },
     };
-    (Octokit as any).mockImplementation(function() { return mockOctokit; });
+    (Octokit as any).mockImplementation(function () {
+      return mockOctokit;
+    });
```

ESLint

[error] Unexpected function expression.

(prefer-arrow-callback)